src/RA_MemManager.h

```diff
@@ -19,6 +19,8 @@ class MemCandidate
 	bool m_bHasChanged;
 };
 
+// TODO: We should try to make these function objects, or a function type, not sure the intention here
+
```

Comment:
These are pointers provided by the emulator to access the memory. They're passed across the DLL boundary as `void*` and cast to `_RAMByteReadFn` and `_RAMByteWriteFn` in `_RA_InstallMemoryBank`.

Reply:
I'm fine with making them `std::function<>` as long as it works with the DLL contract. It shouldn't be part of this PR.

Reply:
I know, that's what the <code>TODO:</code> was for. That'll be for another time.